carbon-design-system/carbon
# feat(react): add unstable_Stack component

Closes #9821

This PR adds in a `Stack` component to be used for vertical spacing along with other components in the design system. This is a useful utility in a component-based model when margin is no longer used by components.

By default, the `Stack` uses a `gap` prop where you can provide a number or a string. If it's a number, it will map to our spacing scale. If it's a string, it will be a custom value and requires a unit.

#### Changelog

**New**

- Add Stack component, related stories, and tests

**Changed**

- Update storybook to include Stack in `carbon-react`

**Removed**

#### Testing / Reviewing

- Verify tests pass as expected
- Verify CSS is included as expected
- Verify story renders as expected
- Look over the API and make sure it meets our API requirements / is understandable
